nvflare/private/fed/server/server_state.py

```diff
@@ -101,14 +106,11 @@ def handle_sd_callback(self, sp: SP, fl_ctx: FLContext) -> ServerState:
 
 
 class Cold2HotState(ServerState):
-    def __init__(self, host: str = "", port: str = "", ssid: str = "") -> None:
-        super().__init__(host, port, ssid)
-
     def register(self, fl_ctx: FLContext) -> dict:
-        return ServerState.IN_SSERVICE
+        return ServerState.IN_SERVICE
 
     def heartbeat(self, fl_ctx: FLContext) -> dict:
-        return ServerState.IN_SSERVICE
+        return ServerState.NOT_IN_SERVICE
```

Comment:
When doing testing, I observe a behavior
:

server is up , 2 clients are up, JOB A is submitted

when I kill the server (for HA), it will persist the snapshot

and then later the server comes up again, BUT before it totally restores its snapshot

client 1 send a heartbeat to server

server say I don't have JOB A, please abort it

client 1 abort it, and then server recover the JOB A from snapshot

the JOB A can never finish

So I think we should block heartbeat service when we are in "Cold2HotState "